docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -874,6 +874,37 @@ As a CDI based runtime, Quarkus extensions often make CDI beans available as par
 However, Quarkus DI solution does not support CDI Portable Extensions.
 Instead, Quarkus extensions can make use of various link:cdi-reference.html[Build Time Extension Points].
 
+=== Extension Health Check
+
+Health check is provided via the `quarkus-smallrye-health` extension. It provides both liveness and readiness checks capabilities.
```

Comment:
"Health check is" to "Health checks are"

Reply:
done